resources/validate_builds.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+set -e
+
+# validated if created build is debug build 
+cd $APK_PATH
+EXPECTED_FLAG='application-debuggable'
+
+for file in *
+do 
+  name=$(echo $file| tail -c 4)
+  if [ $name = 'apk' ]
+  then     
+     echo "Found $file" 
+     APK_NAME=$file
+     break
+  else 
+     echo "Unable to find apk in given directory" 
```

Comment:
^ shouldn't there be an `exit` here? Otherwise, it will continue running the script